An error occurred while executing the following cell:
------------------
import os
import sys
import cv2
import glob
import tqdm
import shutil
import augmax
from typing import Any
from IPython import display
import matplotlib.pyplot as plt
from skimage.util import montage

%matplotlib inline
------------------

---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Input In [3], in <cell line: 3>()
      1 import os
      2 import sys
----> 3 import cv2
      4 import glob
      5 import tqdm

ModuleNotFoundError: No module named 'cv2'
ModuleNotFoundError: No module named 'cv2'
